faq/index.md

```diff
@@ -10,31 +10,101 @@ type: markdown
+
+The code repositories are hosted on [Github](https://github.com/istio). Please see our [Contribution Guidelines](https://github.com/istio/istio/blob/master/CONTRIBUTING.md) to learn how to contribute.
+
+In addition to the code, there are other ways to contribute to the Istio [community](https://istio.io/community/), including on [Stack Overflow](https://stackoverflow.com/questions/tagged/istio), the [mailing list](https://groups.google.com/forum/#!forum/istio-users), and our [Slack channel](https://istio-dev.slack.com/).
 
 #### Where is the documentation?
 
-Check out the [documentation](/docs) right here on istio.io.
+Check out the [documentation](https://istio.io/docs/) right here on istio.io. The docs include [concept overviews](https://istio.io/docs/concepts/), [task guides](https://istio.io/docs/tasks/), [samples](https://istio.io/docs/samples/), and the [complete reference documentation](https://istio.io/docs/reference/).
+
+Detailed developer-level documentation is maintained for each component in github, alongside the code. Please visit each repository for those docs:
+
+*   [Envoy](https://lyft.github.io/envoy/docs/)
+*   [Manager](https://github.com/istio/manager/tree/master/doc)
+*   [Mixer](https://github.com/istio/mixer/tree/master/doc)
+
+#### Istio doesn't work - what do I do?
+
+Follow the [instructions](https://github.com/istio/istio/blob/master/CONTRIBUTING.md#issues) to open an issue [here](https://github.com/istio/istio/issues/new) or ask questions on [slack](https://istio-dev.slack.com/messages/C524NCGR1/). Our [users mailing list](https://groups.google.com/forum/#!forum/istio-users) is another great way to get help and answers. We also monitor [Stack Overflow](https://stackoverflow.com/questions/tagged/istio) for questions tagged with "istio".
+
+Additionally, we provide [Reference Guides](https://istio.io/docs/reference/) for all of the Istio components. These can be helpful when troubleshooting issues with configuration, etc.
+
+#### Why Alpha? And what does the Alpha release cover?
+
+The Alpha release of Istio provides an early preview of the intended functionality and user experience of our service mesh. We are hoping to solicit early feedback on direction and design decisions.
+
+The Alpha release includes the following features:
+
+*   Simple command-line installation into a kubernetes cluster
+*   Scripted proxy injection with traffic capture via iptables
+*   L7 traffic routing rules
+*   In-cluster load balancing for HTTP, GRPC & TCP
+*   Cluster Ingress and Egress
+*   Fault injection
+*   In-memory rate limiting
+*   L7 Metrics and Logs collection
+*   Secure service-to-service authentication with strong identity
+*   Pluggable policy layer and configuration API
+
+#### Does Istio support authn/authz based on attributes in addition to service account (e.g. [Kubernetes Network Policy](https://kubernetes.io/docs/concepts/services-networking/networkpolicies/) based on labels)?
```

Comment:
no abbreviation. Use full forms?
Second, this kind of jumps out of the blue and assumes knowledge of attributes, service accounts, etc.
Can't this whole question be simplified to Does Istio Auth support Authorization? (Eventually yes, but not currently). The bit about mTLS and labels should go into concepts about Istio auth, shouldn't it?